django/utils/six.py

```diff
@@ -873,6 +873,7 @@ def python_2_unicode_compatible(klass):
 if PY3:
     memoryview = memoryview
     buffer_types = (bytes, bytearray, memoryview)
+    from string import ascii_letters
```

Comment:
I think this isn't needed. `from string import ascii_letters` works on Python 2 & 3.